packages/serverpod_serialization/lib/src/websocket_messages.dart

```diff
@@ -6,18 +6,18 @@ import 'package:serverpod_serialization/src/websocket_message_keys.dart';
 /// Base class for messages sent over a WebSocket connection.
 sealed class WebSocketMessage {
   /// The keyword used for the message type in the websocket message.
-  static const String messageTypeKeyword = WebSocketMessageKey.messageTypeKey;
+  static const String _messageTypeKeyword = WebSocketMessageKey.messageTypeKey;
 
   /// The keyword used for the message data in the websocket message.
-  static const String messageDataKeyword = WebSocketMessageKey.messageDataKey;
+  static const String _messageDataKeyword = WebSocketMessageKey.messageDataKey;
```

Comment:
I agree with you halfway here, I think you are right that these two particular definitions are redundant. But I would like to keep the `_messageType` definition for all names since these are references in two quite separated places. 

For example the usage of the `PongCommand._messageType` is both used to identify the message to send it to the correct parser and also when encoding the message.

Reply:
(This might been your intent with the comment as well.)